.gitignore

```diff
@@ -9,3 +9,6 @@
 *.tfvars
 
 **/crash.log
+
+# code editors
+.idea
```

Comment:
This should go into your personal gitignore file to avoid bloat; add something like this to your ~/.gitconfig:

```
[core]
        excludesfile = /Users/shoekstra/.gitignore_global
```

And then add any files/dir to ignore, mine looks like this:

```
.env
tags
```